src/store/error-store.js

```diff
@@ -0,0 +1,22 @@
+import { Store } from 'consus-core/flux';
+
+let error = null;
+
+class ErrorStore extends Store{
+    getError() {
+        return error;
+    }
+
+    clearError(){
```

Comment:
This should be done as an action. See https://github.com/TheFourFifths/consus-client/pull/16#pullrequestreview-7131409